public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.ts

```diff
@@ -208,6 +245,27 @@ export class AzureMonitorQueryCtrl extends QueryCtrl {
     this.target.azureMonitor.metricNamespace = this.target.azureMonitor.metricDefinition;
   }
 
+  migrateApplicationInsightsKeys(): void {
+    const appInsights = this.target.appInsights as any;
+
+    // Migrate old app insights data keys to match other datasources
+    const mappings = {
+      xazis: 'timeColumn',
```

Comment:
Typo here

Reply:
Fixed and added a test to make sure the migration is doing what I expected